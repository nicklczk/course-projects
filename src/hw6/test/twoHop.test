# Tests path finding for two hops

# Create graph
LoadGraph friends champions.tsv

# find path two hops
FindPath friends Kevin Candice