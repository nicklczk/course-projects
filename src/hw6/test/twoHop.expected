# Tests path finding for two hops

# Create graph
loaded graph friends

# find path two hops
path from Kevin to Candice:
Kevin to Eya via Apartments
Eya to Candice via Lander